client/lib/post-normalizer/utils/max-width-photonish-url.js

```diff
@@ -14,8 +14,10 @@ export function maxWidthPhotonishURL( imageURL, width ) {
 	}
 
 	let urlParts = {};
+	let urlType;
 	try {
 		urlParts = getUrlParts( imageURL );
+		urlType = determineUrlType( imageURL );
 	} catch ( e ) {
```

Comment:
do we still need this try/catch? do these methods throw like url.parse did?

Reply:
They don't, that's a good point! 👍